aries-site/src/layouts/content/Example/ExampleControls.js

```diff
@@ -3,11 +3,76 @@ import PropTypes from 'prop-types';
 import { Box, Button, ResponsiveContext } from 'grommet';
 import { Document, Expand, Grommet, Figma } from 'grommet-icons';
 
-export const ExampleControls = ({ designer, docs, figma, setShowLayer }) => {
+export const ExampleControls = ({
+  designer,
+  docs,
+  figma,
+  horizontalLayout,
+  setShowLayer,
+}) => {
   const size = useContext(ResponsiveContext);
   const isSmall = size === 'small';
   const buttonSize = isSmall ? 'small' : undefined;
 
+  if (horizontalLayout) {
+    return (
+      <Box
+        direction={isSmall ? 'column-reverse' : 'row'}
+        align="start"
+        gap="medium"
+        justify="between"
+        pad={{ vertical: 'small' }}
+        round={{ corner: 'bottom', size: 'small' }}
+      >
+        <Box direction="row" wrap>
+          {designer && (
+            <Box flex={false}>
+              <Button
+                title="Open in Grommet Designer"
+                a11yTitle="Open in Grommet Designer"
+                href={designer}
+                icon={<Grommet color="plain" />}
+                target="_blank"
+                size={buttonSize}
+              />
+            </Box>
+          )}
+          {figma && (
+            <Box flex={false}>
+              <Button
+                title="Open in Figma"
+                a11yTitle="Open in Figma"
+                href={figma}
+                icon={<Figma color="plain" />}
+                target="_blank"
+                size={buttonSize}
+              />
+            </Box>
+          )}
+          {docs && (
+            <Box flex={false}>
+              <Button
+                title="Open docs"
+                a11yTitle="Open docs"
+                href={docs}
+                icon={<Document />}
+                target="_blank"
+                size={buttonSize}
+              />
+            </Box>
+          )}
+          <Button
```

Comment:
This seems to be the same as lines 91-125-ish. Any opportunities to simplify/reuse this?

Reply:
Yes, definitely. Implementing on upcoming commit.